site/app/templates/forum/displayThreadList.twig

```diff
@@ -0,0 +1,36 @@
+{% for thread in thread_content %}
+    <a href="{{ thread.link }}">
+        <div class="{{ thread.class }}">
+            {% if thread.pinned == true %}
+                <i class="fas fa-star" style="padding-left:3px;position:relative; float:right; display:inline-block; color:gold; -webkit-text-stroke-width: 1px; -webkit-text-stroke-color: black;" title = "Announcement" aria-hidden="true"></i>
+            {% endif %}
+            {% if thread.favorite == true %}
+                <i class="fas fa-thumbtack" style="padding-left:3px;position:relative; float:right; display:inline-block; color:gold; -webkit-text-stroke-width: 1px; -webkit-text-stroke-color: black;" title="Pinned as my favorite" aria-hidden="true"></i>
+            {% endif %}
+            {% if thread.merged_thread_id != -1 %}
+                <i class="fas fa-link" style="padding-left:3px;position:relative; float:right; display:inline-block; color: white; -webkit-text-stroke-width: 1px; -webkit-text-stroke-color: black;" title="Thread Merged" aria-hidden="true"></i>
+            {% endif %}
+
+            {% if thread.status != 0 %}
+                <i class="fa {{ thread.fa_icon }}" style="margin-right:{{ thread.fa_margin_right }}; padding-left:3px; position:relative; float:right; display:inline-block; color:{{ thread.fa_color }}; font-size:{{ thread.fa_font_size }};" title="{{ thread.tooltip }}" aria-hidden="true"></i>
+            {% endif %}
+
+            {% if thread.is_locked == true %}
+                <i class="fas fa-lock" style="padding-left:3px;position:relative; float:right; display:inline-block; color: white; -webkit-text-stroke-width: 1px; -webkit-text-stroke-color: black;" title="Locked" aria-hidden="true"></i>
+            {% endif %}
+
+            <h4>{{ thread.title | raw  }}</h4>
+            <h5 style="font-weight: normal;">{{ thread.content }}</h5>
+
+            {% for category in thread.categories %}
+                <span class="label_forum" style="background-color: {{ category[1] }}">{{ category[0] }}</span>
+            {% endfor %}
+
+            {% if thread.date.not_null == true %}
+                <h5 style="float:right; font-weight:normal;margin-top:5px">{{ thread.date.formatted }}</h5>
```

Comment:
While we are converting to twig, it might be nice to remove the inline css here. This can possibly be moved into a `forum.css` and we can refer to custom css required for the forum in the stylesheet (maybe a different issue? can discuss).

Reply:
Maybe we can make it a new issue where I can remove all the inline CSS from Forum at one go, will only be dealing with the templates then!